src/main/resources/logback.xml

```diff
@@ -12,9 +12,11 @@
           <pattern>%date [%thread] %-5level %logger{35} - %msg%n</pattern>
         </encoder>
       </appender>
-      <root level="TRACE">
+      <root level="INFO">
```

Comment:
No idea, but since we seem to be assuming that info level stuff makes its way into logs (or at least I do) it seemed incorrect